### ISSUE TITLE
Where can the images used for training in this project be found?

### ISSUE THREAD

**Issue by Sajal-Aggarwal** [carrell-ncl/Windscreen2#4]

**Comment by carrell-ncl** [comment]
Hi,

I havent been able to share then due to privacy. E.g. people's faces abd vehicle registration plates

**Comment by Sajal-Aggarwal** [comment]
> Hi,
> 
> I havent been able to share then due to privacy. E.g. people's faces abd vehicle registration plates

Is there some dataset available that I can use if I am not in the position of acquiring images myself?

**Comment by carrell-ncl** [comment]
Hi,

You can get images for the phone only from Google Open Image dataset. But I recommend only having a small percentage of these,
You'll really need to use images of people actually using their phone, ideally through a windscreen. 90% of these I created myself with many volunteers.
Some images of people using their phone you can take from Google Images and then annotate yourself. 

For the windscreen I would take screenshots from traffic cameras (on Youtube) and then annotate them.

**Comment by Sajal-Aggarwal** [comment]
> to

Ok so if I need to implement  only the first single-step approach using YOLO, then I need 
1. License plate images from the OID dataset
2. Phone images - some from OID and rest being images of people actually using a phone while driving (windscreen) images which I need to procure myself.
Is that right?

**Comment by Sajal-Aggarwal** [comment]
> > to
> 
> Ok so if I need to implement only the first single-step approach using YOLO, then I need
> 
> 1. License plate images from the OID dataset
> 2. Phone images - some from OID and rest being images of people actually using a phone while driving (windscreen) images which I need to procure myself.
>    Is that right?

Also is there any way you could help me out with the windscreen images?

**Comment by carrell-ncl** [comment]
Yeah for the single step you could either use images of the plate or or images of windscreen to count the vehicles.

I can send you images of of the windscreen + annotation XML files.

What's your email address? I will send tomorrow

**Comment by MikuuGithub** [comment]
> Yeah for the single step you could either use images of the plate or or images of windscreen to count the vehicles.
> 
> I can send you images of of the windscreen + annotation XML files.
> 
> What's your email address? I will send tomorrow

Could you send me the file? My email is : 929558605@qq.com. Thank you!

**Comment by moustapha00** [comment]
Hi, 

I discover your work and it interests me a lot, thank you. Would it be possible to send me the images of people using their phone through the windscreen ?

**Comment by LamMoh** [comment]
> Yeah for the single step you could either use images of the plate or or images of windscreen to count the vehicles.
> 
> I can send you images of of the windscreen + annotation XML files.
> 
> What's your email address? I will send tomorrow

Can you please send me the phone dataset as well? this is my email: to_l_k@hotmail.com

I would be much appreciated.

**Comment by Om206** [comment]
hi,
i am omprakash 
i am doing a project, i need dataset
can you send some images or links of images to my email: omprakash.cs19@rvce.edu.in

**Comment by carrell-ncl** [comment]
Hello,
The training images were created especially for this project.
Due to privacy reasons, I cannot share as it contains peoples faces as well as their vehicle licence plates.

BR
Steven

**Comment by Om206** [comment]
from where you found the images, can you plz tell
i am searching for many days and unable to find more images

**Comment by Om206** [comment]
i am studying engineering and need this for my minor project 
i am from india
